src/controllers/DevInteractions.ts

```diff
@@ -0,0 +1,33 @@
+import { ButtonInteraction, Message } from "discord.js";
+import QueueRepository from "../repositories/QueueRepository";
+import ActiveMatchRepository from "../repositories/ActiveMatchRepository";
+import MessageBuilder, { ButtonCustomID } from "../utils/MessageBuilder";
+import { fillTeams } from "../utils/devFillTeams";
+
+export async function handleDevInteraction(buttonInteraction: ButtonInteraction): Promise<void> {
+  const message = buttonInteraction.message;
+  if (!(message instanceof Message)) return;
+
+  switch (buttonInteraction.customId) {
+    case ButtonCustomID.FillTeam: {
+      const fullTeam = await fillTeams();
+
+      await message.edit(MessageBuilder.fullQueueMessage(fullTeam));
+      break;
+    }
+    case ButtonCustomID.RemoveAll: {
+      await QueueRepository.removeAllBallChasersFromQueue();
+
+      const ballChasers = await QueueRepository.getAllBallChasersInQueue();
+      await message.edit(MessageBuilder.queueMessage(ballChasers));
+      break;
+    }
+    case ButtonCustomID.BreakMatch: {
+      await ActiveMatchRepository.removeAllPlayersInActiveMatch("346838372649795595");
+
+      const ballChasers = await QueueRepository.getAllBallChasersInQueue();
+      await message.edit(MessageBuilder.queueMessage(ballChasers));
```

Comment:
Break match should just remove the message since the queue embed is already sent. No need to edit. I know it's only a dev command, but I imagine we'll use similar logic when we implement broken queue so want to make sure it's clear.

Reply:
Fixed.